### PR TITLE
Cms

### PR DESCRIPTION
**Pull request by f-gate**
Update content to be reviewed and the login.

### CODE REVIEW

File: about.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+require 'dbConn.php';
+session_start();
+if (!isset($_SESSION['admin']) || $_SESSION['admin'] != 'loggedIn') {
+    header('Location: login.php');
+}
+
+function retrieve_data($db) {
+    $retrieve_statement = $db->query('SELECT `content`, `picture_link`, `phone_number`, `email` FROM `about_me` WHERE `id` = 1;');
+    $start_data = $retrieve_statement->fetch();
```
Comment (mporam) [inline about.php:10]: you can just do `return $retrieve_statement->fetch();`, you dont need the extra variable

File: db_querys_products.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+           $query->bindParam(4, $code_link);
+           $query->bindParam(5, $selector);
+           $query->execute();
+           $row_count = $query->rowCount();
+           return "Replaced $row_count row(s). \n";
+
+}
+
+$position = $_POST['position'];
+echo set_projects($picture_link, $picture_text, $db, $picture_image_front, $code_link, $position);
```
Comment (mporam) [inline db_querys_products.php:41]: put this inside your html body

File: initialscreen.php [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+    $retrieve_statement->execute();
+    $start_data = $retrieve_statement->fetch();
+    return $start_data;
+}
+
+$start_data = retrieve_data($db);
+
+function retrieve_data_project($db, $selector) {
+
+    $retrieve_statement = $db->prepare('SELECT `picture_link`, `picture_text`, `picture_image_front`, `code_link` FROM `projects` WHERE `id` = ?;');
```
Comment (mporam) [inline initialscreen.php:15]: if you dont set an ID and use a fetchAll, you can get all 9 in one function call

File: initialscreen.php [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+            echo $start_data['content']
+            ?>
+        </p>
+    </section>
+    <!--PORTFOLIO CONTENT    THIS IS WHERE THE PICTURE DATA WILL GO NEED TO BUILD HTML FOR THIS   -->
+    <div id="project"></div>
+    <section class="portfolio"  >
+        <h2 class="project" > Projects </h2>
+        <div class="boxholder">
+            <div class="picturelink">
```
Comment (mporam) [inline initialscreen.php:113]: if you select the data using fetch all as mentioned above, you can do this in a loop

**Review by mporam — CHANGES_REQUESTED** [review]

File: about.php (hunk 1 above)
Comment (f-gate) [reply about.php:10]: done

**Pull request closed without merging**